scripts/footprint/plan.txt

```diff
@@ -10,3 +10,7 @@ footprints,default,ehl_crb,tests/benchmarks/footprints,
 footprints,userspace,ehl_crb,tests/benchmarks/footprints,-DCONF_FILE=prj_userspace.conf
 echo_client,default,frdm_k64f,samples/net/sockets/echo_client,
 echo_server,default,frdm_k64f,samples/net/sockets/echo_server,
+bt_beacon, default, nrf52840dk_nrf52840, samples/bluetooth/beacon
+bt_peripheral, default, nrf52840dk_nrf52840, samples/bluetooth/peripheral
+bt_central, default, nrf52840dk_nrf52840, samples/bluetooth/central
+mesh_demo, default, bbc_microbit, samples/bluetooth/mesh_demo
```

Comment:
Should that not be bt_mesh_demo? Such that it matches those above.

Reply:
```suggestion
bt_mesh_demo, default, bbc_microbit, samples/bluetooth/mesh_demo
```